### PR TITLE
Yet another MainMenu improvements PR

### PR DESCRIPTION
**Pull request by MoNTE48**

### CODE REVIEW

File: src/client/game.cpp [hunk 1]
```diff
@@ -4359,15 +4359,15 @@ void Game::showPauseMenu()
 #if __IOS__
 	ypos += 0.5f;
 #endif
-	std::ostringstream os;
+	std::ostringstream os; // ToDo: ".x2" for buttons
```
Comment (MoNTE48) [inline src/client/game.cpp:4362]: @luk3yx any idea how is best way to  implement this?
```lua
local high_dpi = (PLATFORM == "OSX" and tonumber(core.settings:get("screen_dpi")) / 72 >= 2) or
		core.get_screen_info().density >= 3
```

Comment (luk3yx) [reply src/client/game.cpp]: I've added a `Game::isHighDpi()` method

**Comment by MoNTE48** [comment]
@luk3yx, what about 'StyleSpec.h'?

File: builtin/mainmenu/dlg_contentstore.lua [hunk 2]
```diff
@@ -367,9 +370,10 @@ function install_dialog.get_formspec()
 
 	local formspec = {
 		"formspec_version[3]",
-		"size[7,7.85]",
+		"size[7,7.85;false]",
```
[the remark this answers is not in the document]
Comment (luk3yx) [reply builtin/mainmenu/dlg_contentstore.lua:373]: It's the default value

**Pull request merged**